src/parse.rs

```diff
@@ -293,7 +295,7 @@ pub(crate) async fn handle_unilateral(
         Response::MailboxData(MailboxDatum::Status { mailbox, status }) => {
             unsolicited
                 .send(UnsolicitedResponse::Status {
-                    mailbox: (*mailbox).into(),
+                    mailbox: (mailbox.as_ref()).into(),
                     attributes: status
                         .iter()
                         .map(|s| match s {
```

Comment:
can be replaced by a real clone once https://github.com/djc/tokio-imap/pull/125 is merged